packages/backend/src/config/getDeliveryServiceProperties.ts

```diff
@@ -31,7 +32,14 @@ export function getDeliveryServiceProperties(
         //The interface DeliveryServiceProperties requires all properties to be non-null. But since we are accepting a partially filled config.yml we are overwriting the required fields so basically no property is required at all. This can be done because every missing property is replaced by a default property
         {
             ...schema.DeliveryServiceProperties,
-            required: [],
+            definitions: {
```

Comment:
Normally the auto-generated is based on a TS type. Couldn't we do this here too?

Reply:
The schema is already auto-generated via TS. However, the TS type requires every value to be required. This will be the case after the config reader has finished, but when reading the config every type is optional because when not specified additionally it'll be substituted with the default value for that property.